.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+<!-- Thank you for contributing! -->
+
+## Description
+
+<!-- Please insert your description here and provide especially info about the "what" this PR is solving -->
+
+### Additional context
+
+<!-- e.g. is there anything you'd like reviewers to focus on? -->
+
+### Issue reference
+
+<!-- Please insert a link to the solved issue. If none, create one for this PR and then reference it here -->
+
+---
+
+### What is the purpose of this pull request? <!-- (put an "X" next to an item) -->
```

Comment:
Do we really need this? I thought this should be obvious from the commit prefix and/or the labels.

Reply:
You are right, this duplicates that thing.